rotkehlchen/db/dbhandler.py

```diff
@@ -1286,49 +1270,51 @@ def get_yearn_vaults_events(
     ) -> List[YearnVaultEvent]:
         cursor = self.conn.cursor()
         query = cursor.execute(
-            'SELECT '
-            'event_type, '
-            'from_asset, '
-            'from_amount, '
-            'from_usd_value, '
-            'to_asset, '
-            'to_amount, '
-            'to_usd_value, '
-            'pnl_amount, '
-            'pnl_usd_value, '
-            'block_number, '
-            'timestamp, '
-            'tx_hash, '
-            'log_index '
-            'from yearn_vaults_events WHERE address=? AND (from_asset=? OR from_asset=?);',
+            'SELECT * from yearn_vaults_events WHERE address=? '
+            'AND (from_asset=? OR from_asset=?);',
             (address, vault.underlying_token.identifier, vault.token.identifier),
         )
         events = []
         for result in query:
-            realized_pnl = None
-            if result[7] is not None:
-                realized_pnl = Balance(amount=FVal(result[7]), usd_value=FVal(result[8]))
-            events.append(YearnVaultEvent(
-                event_type=result[0],
-                from_asset=Asset(result[1]),
-                from_value=Balance(amount=FVal(result[2]), usd_value=FVal(result[3])),
-                to_asset=Asset(result[4]),
-                to_value=Balance(amount=FVal(result[5]), usd_value=FVal(result[6])),
-                realized_pnl=realized_pnl,
-                block_number=int(result[9]),
-                timestamp=Timestamp(int(result[10])),
-                tx_hash=result[11],
-                log_index=result[12],
-            ))
+            try:
+                events.append(YearnVaultEvent.deserialize_from_db(result))
+            except DeserializationError as e:
+                msg = f'Failed to read yearn vault event from database {result}'
+                self.msg_aggregator.add_warning(msg)
+                log.error(msg, error=str(e))
```

Comment:
Here you should do the opposite. The smaller message should go to the aggregator, as the aggregator messages are user facing.

And longest one in the logs. Also ... use same verbosity in aggregator and logs.

```suggestion
                msg = f'Failed to read yearn vault event from database due to {str(e)}'
                self.msg_aggregator.add_warning(msg)
                log.warning(msg, data=result)
```